src/main/java/org/spongepowered/api/block/data/CommandBlock.java

```diff
@@ -24,14 +24,15 @@
  */
 package org.spongepowered.api.block.data;
 
-import com.google.common.base.Optional;
 import org.spongepowered.api.text.message.Message;
-import org.spongepowered.api.util.command.source.CommandBlockSource;
+import org.spongepowered.api.util.command.CommandSource;
+
+import com.google.common.base.Optional;
 
 /**
  * Represents a Command Block.
  */
-public interface CommandBlock extends TileEntity, CommandBlockSource {
+public interface CommandBlock extends TileEntity, CommandSource {
```

Comment:
Whoops... Did the name of CommandBlockSource change? For some reason I thought it changed to CommandSource, but that's not right.